packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -485,7 +489,17 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
       case 1:
         widget.focusNode.requestFocus();
         hideToolbar();
-        _clearSelection();
+        switch (defaultTargetPlatform) {
+          case TargetPlatform.android:
+          case TargetPlatform.fuchsia:
+          case TargetPlatform.iOS:
+            // On mobile platforms the selection is set on tap up.
```

Comment:
Is this really a platform thing or is it a mouse vs. tap thing?

Reply:
Yeah this is a platform thing it seems like. I tried my iPad with the trackpad and keyboard and it doesn't set the selection until tap up. I also tried this on my pixel 7 with a mouse with the same results.

Reply:
Alright, thank you for checking!